### ISSUE TITLE
Refractor PlayerScreen

### ISSUE THREAD

**Issue by jbkuczma** [jbkuczma/NBAreact#54]
The PlayerScreen can now be reached 3 different ways:
- From the roster
- From the league leaders
- From the player search

PlayerScreen is currently written for data returned from the roster API call, but LeagueLeaders and PlayerSearch API calls return different properties.

| Property  | Roster | Player Search | League Leader |
| ------------- | ------------- | ------------- |-|
| Player ID | player_id  | personId | player_id  |
| Name | player | firstName & lastName | player |
| Height  | height  | heightFeet & heightInches | |
| Weight | weight | weightPounds | |
| Years Experience | exp | yearsPro | |
| School | school | collegeName | |
| Number | num | jersey | |

Based on this, if we are navigating from a league leader item to the player screen, the player will have to be looked up from PlayersInLeague array from the Redux Store and that value will be used.

tl;dr Refractor PlayerScreen to use data for a player found in the PlayersInLeague array
